admiral/pkg/controller/common/types.go

```diff
@@ -22,7 +22,7 @@ type SidecarEgress struct {
 }
 
 type SidecarEgressMap struct {
-	cache map[string][]SidecarEgress
```

Comment:
It looks to me like this is now mapping the identity value -> a map of namespace -> sidecar egress object. Can you add a comment to make that clear?

Reply:
Done